### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by fercostacastro**

### CODE REVIEW

**Comment by manuelbarzi** [comment]
change the base branch to develop

**Comment by manuelbarzi** [comment]
> change the base branch to develop

i already did it

File: staff/fernando-costa/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+see the path fo the files
+```bash
+```
+## rm -rf 
+remove all folders and its contents
+```bash
+$ rm -rf workspace/hello world
+```
+
+# git branch -D
```
Comment (Barrufat) [inline staff/fernando-costa/bash/README.md]: Git commands should not be included in this file. If you want, you can create another branch with your git-cheat-sheet file🧐

File: staff/fernando-costa/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+## mkdir (make directory)
+it creats a new directory
+```bash
+$ mkdir
+```
+
+## cd
+it changes a directory
+```bash
+$ cd
```
Comment (manuelbarzi) [inline staff/fernando-costa/bash/README.md]: add a target directory in the example

Comment (fercostacastro) [reply staff/fernando-costa/bash/README.md]: example added

File: staff/fernando-costa/bash/README.md (hunk 1 above)
Comment (fercostacastro) [reply staff/fernando-costa/bash/README.md]: All Git commands have been removed!

File: staff/fernando-costa/bash/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+## ls -l
+all folders we have, but with more info
+```bash
+$ ls -l
+```
+
+## touch    
+it creats a file i.e. hello.world
+```bash
+$ touch
```
Comment (Barrufat) [inline staff/fernando-costa/bash/README.md]: The name of the file that you want to create should appear next to the "touch" comand 😶

File: staff/fernando-costa/bash/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+## ls -a
+hidden folders
+```bash
+$ ls -a
+```
+
+## mkdir (make directory)
+it creats a new directory
+```bash
+$ mkdir
```
Comment (Barrufat) [inline staff/fernando-costa/bash/README.md]: The name of the folder that you want to create should appear next to the "mkdir" comand 😗

File: staff/fernando-costa/bash/README.md (hunk 3 above)
Comment (fercostacastro) [reply staff/fernando-costa/bash/README.md]: It's done now :)

File: staff/fernando-costa/bash/README.md (hunk 4 above)
Comment (fercostacastro) [reply staff/fernando-costa/bash/README.md]: It's done now :)

**Pull request merged**